reference-architecture/aws-ansible/playbooks/roles/prerequisites/tasks/main.yaml

```diff
@@ -11,3 +11,4 @@
       - bash-completion
       - docker
       - python-httplib2
+  when: ansible_pkg_mgr != "unknown" 
```

Comment:
If this requires ansible 2.x+, you can use the `package` module rather than yum.